app/helpers/schema_helper.rb

```diff
@@ -2,14 +2,7 @@ module SchemaHelper
   FORM_RESPONSE_SCHEMA = File.read(Rails.root.join("config/schemas/form_response.json"))
 
   def validate_against_form_response_schema(data)
-    data = data.deep_stringify_keys
     schema = JSON.parse(FORM_RESPONSE_SCHEMA)
-    validator = JSONSchemer.schema(schema)
-    errors = validator.validate(data).to_a
-
-    OpenStruct.new(
-      fail: errors.any?,
-      errors: errors.map { |report| report["details"] },
-    )
+    JsonValidator.new schema, data
```

Comment:
```suggestion
    JsonValidator.new(schema, data)
```

This is much nicer to read!